kickoff/test/js/tests.js

```diff
@@ -90,15 +90,22 @@ assert.strictEqual( guessBranchFromVersion("firefox", "31.0.1esr"), "releases/mo
 QUnit.test( "addLastVersionAsPartial", function( assert ) {
     name="firefox";
     base = getBaseRepository(name);
-    previousBuilds = {"releases/mozilla-beta": ["31.0b2build2", "30.0b9build2", "29.0b10build2", "25.0b5build2", ],
-                      "releases/mozilla-release": ["33.0.1build2", "32.0.1build2",  "28.0build2", "27.0build2"],
-                      "releases/mozilla-esr31": ["31.1.0esrbuild1", "29.4.0esrbuild1", "29.2.0esrbuild1", "24.3.0esrbuild1" ]};
+    previousBuilds = {
+        "releases/mozilla-beta": [
+            "31.0b2build2", "30.0b9build2", "29.0b10build2", "25.0b5build2",
+            // Test data for human sort. See bug 1289627.
+            "48.0b1build2","47.0b9build1","47.0b8build1", "48.0b9build1", "48.0b7build1", "48.0b6build1"
+        ],
+        "releases/mozilla-release": ["33.0.1build2", "32.0.1build2",  "28.0build2", "27.0build2"],
+        "releases/mozilla-esr31": ["31.1.0esrbuild1", "29.4.0esrbuild1", "29.2.0esrbuild1", "24.3.0esrbuild1" ]
```

Comment:
Sorry, I'm not sure where I dealt with Thunderbird specifically 😕 What kind of tests do you want me to add?

For the record, [these changes](https://github.com/mozilla-releng/ship-it/pull/88/files#diff-f2f19f23c632c1a8154fd734bebd6ed1L65) are a small refactor, as we fetch the same value as an ESR. I believe we already have tests for it in `guessBranchFromVersion()`.

[This regex](https://github.com/mozilla-releng/ship-it/pull/88/files#diff-ca14b13e931fac8ec38266fd66bfa441R7) is just a carry over of the one [declared there](https://github.com/mozilla-releng/ship-it/pull/88/files#diff-f2f19f23c632c1a8154fd734bebd6ed1L5). I can move it back in `suggestion.js`, if it makes things clearer.